docs/aegis/roadmap/sbom.md

```diff
@@ -0,0 +1,54 @@
+---
+title: SBoM
+area: Supply Chain
+status: Planning
+funding_required: true
+index: 13
+previous:
+  url: hex-asset-side-loading
+  title: Hex Asset Side-Loading
+---
+
+## Goals
+
+SBoM Support in the default toolchain and integration into common industry
+Tooling
+
+## Impact
+
+By integrating Software Bill of Materials (SBoM) support into the default Erlang
+ecosystem toolchain and aligning with industry standards (SPDX 3.0.1,
+CycloneDX 1.6), this milestone significantly increases visibility and
+traceability of dependencies across source, build, runtime, and cryptographic
+layers. By offering plugins for common build tools (mix, rebar3, Gleam),
+integrating with scanning solutions (ORT, ScanCode), and storing SBoM data at
+both compile and runtime, developers gain immediate introspection and
+verification capabilities—even for foreign dependencies like NIFs or JS bundles.
+Extending these practices to the core infrastructure (Hex package manager,
+official Docker images, and key language projects) ensures a complete, auditable
+record of all software components, fostering stronger security, compliance, and
+trust within the Erlang and Elixir communities.
+
+## Deliverables
+
+* Integrations into ORT (mix, rebar3, Gleam)
+* Integrations into ScanCode (mix, rebar3, Gleam)
+* Build Tools (or plugins for build tools) can generate SBoMs
+  - Type: Source, Build, Runtime, & Cryptography
+  - Formats: SPDX v3, CycloneDX 1.6
```

Comment:
SPDX 2,2 is the only ISO (AFAIK).
When SPDX 3.X is an ISO, then yes.

Reply:
Let's make our life simple and not mention the version for now. Then we can decide what makes sense at the given time.